azext_iot/central/params.py

```diff
@@ -99,6 +106,8 @@ def load_central_arguments(self, _):
         context.argument(
             "module_id", options_list=["--module-id", "-m"], help="Iot Edge Module ID",
         )
+
+    with self.argument_context("iot central app user") as context:
         context.argument(
             "assignee",
             options_list=["--user-id", "--assignee"],
```

Comment:
@digimaun @valluriraj  how do you feel about removing the individual "ids" we have everywhere (user-id, export-id, token-id, etc) and combining them all under a single "--id" and "-i" flag?

Reply:
Can you describe a modified command example to make sure we understand the proposal? Here are my initial thoughts:

I am not a fan of using plain "--id" because "--id" or "--ids" typically refer to fully qualified Azure resource id's. Also some of the central commands, such as user creation have lots of *-id arguments. For example user create could potentially have 4 different required Id's.  If the idea is cut down to a single arg that is expecting json, I think that would increase usage difficulty by forcing users to deal with json. Also a user may find it easier to understand a command that has separated arguments in this category from the discoverability of documentation/--help per argument.

Reply:
Agree with using separate identifiers